### PR TITLE
feat(angular): support extra sign-in params in angular

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Support extra sign-in params in angular

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/js/src/utils/angular.ts [hunk 1]
```diff
@@ -52,6 +54,32 @@ export type LogtoAngularConfig = {
    * @default true
    */
   includeReservedScopes?: boolean;
+  /**
+   * Login hint indicates the current user (usually an email address or a phone number).
+   * @link SignInUriParameters.loginHint
+   */
+  loginHint?: SignInUriParameters['loginHint'];
```
Comment (wangsijie) [inline packages/js/src/utils/angular.ts:61]: I’m just curious, why is there a special parameter file for Angular?

**Review by wangsijie — APPROVED** [review]

File: packages/js/src/utils/angular.ts (hunk 1 above)
Comment (charIeszhao) [reply packages/js/src/utils/angular.ts:61]: It's used for converting Logto parameters into the ones supported by `augular-auth-oidc-client` library, as we don't have a native Angular SDK.

File: packages/js/src/utils/angular.ts [hunk 2]
```diff
@@ -52,6 +54,32 @@ export type LogtoAngularConfig = {
...
+  /**
+   * The first screen to be shown in the sign-in experience.
+   * @link SignInUriParameters.firstScreen
+   */
+  firstScreen?: SignInUriParameters['firstScreen'];
+  /**
+   * Identifiers used in the identifier sign-in or identifier register page.
```
Comment (xiaoyijun) [inline packages/js/src/utils/angular.ts:68]: Also reset password page

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

File: packages/js/src/core/sign-in.ts [hunk 3]
```diff
@@ -34,9 +34,9 @@ export type SignInUriParameters = {
    */
   firstScreen?: FirstScreen;
   /**
-   * Specifies identifiers used in the identifier sign-in or identifier register page.
+   * Specifies identifiers used in the identifier sign-in, register, and reset password pages.
```
Comment (xiaoyijun) [inline packages/js/src/core/sign-in.ts:37]:
```suggestion
   * Specifies identifiers used in the identifier sign-in, identifier register, and reset password pages.
```

File: packages/js/src/core/sign-in.ts [hunk 4]
```diff
@@ -34,9 +34,9 @@ export type SignInUriParameters = {
...
    *
-   * Available values: `email`, `phone`, `username`.
+   * Available values: `email`, `phone`, or `username`.
```
Comment (xiaoyijun) [inline packages/js/src/core/sign-in.ts:39]: No `or`. since they can be used together.

Comment (charIeszhao) [reply packages/js/src/core/sign-in.ts:39]: OK

**Pull request merged**